docs/source/starters/index.md

```diff
@@ -32,7 +32,7 @@ The configuration file must contain:
 
 Additionally, the configuration file may contain:
 
-* `tools`: The tools to customise your project setup with. Select from comma-separated values, ranges, or 'all/none'. Omitting this from your configuration file will result in the default selection of `none`.
+* `tools`: The tools to customise your project setup with. Select from comma-separated values `lint, test, log, docs, data, pyspark, viz` or `all/none`. Omitting this from your configuration file will result in the default selection of `none`.
 * `example`: Indicate `yes` or `no` to select whether you would like your project to be populated with example code. Omitting this from your configuration file will result in the default selection of `no`.
```

Comment:
This is `example_pipeline` still, isn't it?

Reply:
Thank you, @ankatiyar , fixed!